File: HealthSysHub.Web.UI/Models/Hospital.cs
namespace HealthSysHub.Web.UI.Models
{
    public class Hospital
    {
        public Guid? HospitalId { get; set; }
        public string? HospitalName { get; set; }
        public string? HospitalCode { get; set; }
        public string? RegistrationNumber { get; set; }
        public string? Address { get; set; }
        public string? City { get; set; }
        public string? State { get; set; }
        public string? Country { get; set; }
        public string? PostalCode { get; set; }
        public string? PhoneNumber { get; set; }
        public string? Email { get; set; }
        public string? Website { get; set; }
        public string? LogoUrl { get; set; }
        public Guid? HospitalTypeId { get; set; }
        public Guid? CreatedBy { get; set; }
        public DateTimeOffset? CreatedOn { get; set; }
        public Guid? ModifiedBy { get; set; }
        public DateTimeOffset? ModifiedOn { get; set; }
        public bool IsActive { get; set; }
    }
}

File: HealthSysHub.Web.Pharmacy.UI/src/app/models/hospital.ts
export interface Hospital {
  hospitalId: string;
  hospitalName?: string | null;
  hospitalCode?: string | null;
  registrationNumber?: string | null;
  address?: string | null;
  city?: string | null;
  state?: string | null;
  country?: string | null;
  postalCode?: string | null;
  phoneNumber?: string | null;
  email?: string | null;
  website?: string | null;
  logoUrl?: string | null;
  hospitalTypeId?: string | null;
  createdBy?: string | null;
  createdOn?: Date | string | null;
  modifiedBy?: string | null;
  modifiedOn?: Date | string | null;
  isActive: boolean;
}
